<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationListView</class>
 <widget class="QWidget" name="NavigationListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QEXTNavigationListView" name="navListView1"/>
   </item>
   <item row="0" column="3">
    <widget class="QEXTNavigationListView" name="navListView4"/>
   </item>
   <item row="0" column="1">
    <widget class="QEXTNavigationListView" name="navListView2"/>
   </item>
   <item row="0" column="2">
    <widget class="QEXTNavigationListView" name="navListView3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEXTNavigationListView</class>
   <extends>QListView</extends>
   <header>qextNavigationListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
